docs/en/serverless/quickstarts/collect-data-with-native-otlp.asciidoc

```diff
@@ -0,0 +1,208 @@
+[[collect-data-with-native-otlp]]
+= Quickstart: Collect data with Elastic Cloud Native OpenTelemetry Endpoint
+
+preview::[]
+
+In this quickstart guide, you'll learn how to use the Elastic Cloud Native OpenTelemetry Endpoint to send logs, metrics, and traces to Elastic.
+
+[discrete]
+== What is the Elastic Cloud Native OpenTelemetry Endpoint?
+
+The Elastic Cloud Native OpenTelemetry Endpoint is a fully managed offering exclusively for Elastic Cloud Serverless users that simplifies OpenTelemetry data ingestion. It provides an endpoint for OpenTelemetry SDKs and Collectors to send telemetry data, with Elastic handling scaling, data processing, and storage.
+
+This endpoint is designed for the following use cases:
+
+* Logs & Infrastructure Monitoring: Logs forwarded in OTLP format and host and Kubernetes metrics in OTLP format.
+* APM: Application telemetry in OTLP format.
+
+[%collapsible]
+.Differences from the existing Elastic OTLP Endpoint
+====
+The Elastic Cloud Native OpenTelemetry Endpoint ensures that OpenTelemetry data is stored without any schema translation, preserving both semantic conventions and resource attributes. It supports ingesting OTLP logs, metrics, and traces in a unified manner, ensuring consistent treatment across all telemetry data. This marks a significant improvement over the {observability-guide}/apm-open-telemetry.html[existing functionality], which primarily focuses on traces and the APM use case.
```

Comment:
resource attributes is part of semantic convention. Maybe reword it as `Opentelemetry semantic conventions`?

Reply:
updated